sklearn/metrics/_plot/roc_curve.py

```diff
@@ -138,6 +153,21 @@ def plot_roc_curve(estimator, X, y, pos_label=None, sample_weight=None,
     -------
     display : :class:`~sklearn.metrics.RocCurveDisplay`
         Object that stores computed values.
+
+    Examples
+    --------
+    >>> import pytest
+    >>> plt = pytest.importorskip('matplotlib.pyplot')
+    >>> from sklearn import datasets, metrics, model_selection, svm
+    >>> X, y = datasets.make_classification(random_state=0)
+    >>> X_train, X_test, y_train, y_test = model_selection.train_test_split(\
+            X, y, random_state=0)
+    >>> clf = svm.SVC(random_state=0)
+    >>> clf.fit(X_train, y_train)
+    SVC(random_state=0)
+    >>> metrics.plot_roc_curve(estimator=clf, X=X_test, y=y_test)
```

Comment:
I am happy with skipping this and removing the `pytest.importorskip`. Since `estimator`, `X`, `y` are position arguments, I would be okay with using positions here.